task_03.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A useful docstring."""
+
+import decimal
+
+NAME = raw_input('What is your name? ')
+P = raw_input('What is the anmount of your principle? ')
+T = raw_input('For how many years is the loan being borrowed? ')
+PREQ = raw_input('Are you pre-qualified for this loan? ')
+YES_PREQ = PREQ == 'Yes' or PREQ == 'y'
+NO_PREQ = PREQ == 'No' or PREQ == 'n'
+P = int(P)
+T = int(T)
+N = 12
+
+
+def do_report(name, prin, years, preq, total):
+    """Report Function.
+
+    This function generates a report for the compounded interest fees.
+
+    Args:
+        name (str): Name of the customer
+        prin (int): Principle
+        years (int): Duration of loan
+        preq (str): Pre-qualified status
+        total (float): Total loan cost (principle + interest)
+
+    Returns:
+        report (str): A formatted report for the customer.
+
+    Example:
+        What is your name? Joe
+        What is the anmount of your principle? 299334
+        For how many years is the loan being borrowed? 20
+        Are you pre-qualified for this loan? Yes
+        Loan Report for: Joe
+        --------------------------------------------------
+        Principle:                    $299,334
+        Duration:                     20yrs
+        Pre-qualified?:               Yes
+
+        Total:                        $575,172.817
+"""
+    report = 'Loan Report for: {}\n'.format(name)
+    report += '-' * 50 + '\n'
+    report += '\t{:<15} {:>15}{:,}\n'.format('Principle:', '$', prin)
+    report += '\t{:<15} {:>15}{:>}\n'.format('Duration:', years, 'yrs')
+    report += '\t{:<15} {:>15}\n\n'.format('Pre-qualified?:', preq)
+    if TOTAL is None:
+        report += '\t{:<15} {:>15}\n'.format('Total:', total)
+    else:
+        report += '\t{:<15} {:>15}{:,}\n'.format('Total:', '$', total)
+    return report
+
+
+if P >= 0 and P <= 199999:
+    if T >= 1 and T <= 15:
+        if YES_PREQ:
+            R = 3.63
```

Comment:
You ran aground of a floating point issue. Here you've defined the interest rate as floating points, not as strings (for later conversion to decimals). This ruined your precision later which was only noticeable with _really big_ numbers.